app/src/atoms/InputField/index.tsx

```diff
@@ -306,6 +303,11 @@ function Input(props: InputFieldProps): JSX.Element {
           {props.secondaryCaption}
         </StyledText>
       ) : null}
+      {props.error != null ? (
```

Comment:
this component has `const error = props.error != null` so we can use error directly, can't we?

```suggestion
      {error ? (
```